app/search_builders/sufia_search_builder.rb

```diff
@@ -0,0 +1,9 @@
+class SufiaSearchBuilder < Blacklight::Solr::SearchBuilder
```

Comment:
@mjgiarlo tried it, and realized the other one is a module.  I want a search builder I can instantiate so that I do not get the other catalog controller permissions mixed in.